packages/website/tsconfig.json

```diff
@@ -11,5 +11,5 @@
 			"~/*": ["./src/*"]
 		}
 	},
-	"include": ["src/**/*.ts"]
+	"include": ["src/**/*.ts", "src/model.server.tsx"]
```

Comment:
```suggestion
	"include": ["src/**/*.ts", "src/**/*.tsx"]
```

Reply:
Asides from this, the [remix.env.d.ts](https://github.com/discordjs/discord.js/blob/main/packages/website/remix.env.d.ts) and [types.d.ts](https://github.com/discordjs/discord.js/blob/main/packages/website/types.d.ts) files on the website root should probably be included too